### DISCUSSION TITLE
幫忙改拼字文法的「小軟體」估值衝破 130 億美元，變身美國最值錢公司之一

### DISCUSSION THREAD

**Discussion by ruthchen1020** [1101-GS4719-C/Python-Community-GS4719-C#886] [General]
AI寫作工具Grammarly是學生用來改論文、打工者寫信、網友發文的工具。這個Chrome擴展程式會在打字時幫忙改拼字和文法，也能根據文章或訊息發送的對象選擇合適的語氣語調。如寫信給老闆或教授時，可選擇「Formal」風格，寫給家人朋友時選擇「Friendly」風格等，AI 可以根據語調特徵給予合適單字和語氣建議。我覺得AI真的是一個人類偷懶的工具。
https://cdn.technews.tw/2021/12/05/grammarly-valuation/

**Comment by StevenWanglolz** [comment]
沒想到一個幫忙修改英文拼字和文法的小軟體也可以這麼成功! 而且這還不是由英文母語者開發的! 生活中處處都充滿著AI的技術啊

**Comment by Chiway1229** [comment]
原來有這麼一樣的軟體，過去從沒聽說過，看來可以下載來試用看看了

**Comment by bonny123865** [comment]
Grammarly真的很好用，剛剛看文章中有介紹到可以切換風格
原本想說是甚麼新功能，原來是需要訂閱的，我還是當個免費仔就好!

**Comment by Kotori0501** [comment]
沒想到打個英文文章，還可以依照寫給特定的人物而去改變英文單字，感覺上還不錯用。
但我比較好奇一點，如果是用Google翻譯的英文會不會可以轉換成比較順的英文語法

**Comment by dabbie109303501** [comment]
這個我知道! 在youtube上廣告打很兇，但還蠻好用的，像是要用英文演講的稿子就可以直接在上面打，它可以幫忙揪出文法錯誤，但如果想讓文章變得十分優美就要付費了

**Comment by yuxinlin93** [comment]
grammerly的廣告真的是到處都是 " writing is not that easy. But Grammarly can help. This scentence is grammatically correct, but it's wordy." 連英文課老師都要求我們在交作業以前先用grammarly檢查。除可以檢查出拼寫錯誤、文法錯誤，還會推薦一些可以更換的字句，讓文章變得更通順、優美，真的是一個很優質的app

**Comment by CPCpyt** [comment]
很常看到他的廣告，沒想到背後有這樣的故事，
感謝分享！獲益良多！

**Comment by Doreen331** [comment]
之前被grammerly的廣告打到有點心動，剛好有使用需求就有使用過，依照我個人的使用經驗，還是覺得靠自己會比較實在一點，grammerly有時候還是會無法偵測到我想要使用的句型，反而會增加我使用上的困擾，但如果是要寫沒有華麗句型的文章，或是單純想要練習英文，grammerly還是一個不錯的選擇～

**Comment by wenchi9023** [comment]
以前就知道grammerly可以改錯、修改不合適的字詞
但今天我才知道它還可以根據不同情境替換語氣，太方便了!這會是我進一步購買的因素!

**Comment by dingxue9080** [comment]
感覺相當實用 一般可能只會在乎語意或文法上順不順 但能針對書寫的對象做語氣上的更改很厲害！AI的應用真的好廣泛

**Comment by Floriaand** [comment]
居然可以根據情境替換語氣，太厲害了，感覺我可以多跟它學說話，我好像一直講話都太直接。

**Comment by Zoe06060606** [comment]
我也有用過！
這個對於文法不好的人真的是一大救星
以前都會拿這個來初步修正英文作文

**Comment by dinosaur1009** [comment]
這個感覺在需要寫一些平時不常用的文章很實用！

**Comment by swerfg03joe** [comment]
這個小軟體確實很實用，只要將文章輸入進去就能幫我抓出錯誤文法或更加合適的寫法，不過其價值能否高達百億美元我認為是不值得，但美國市場就是這樣，很多新創公司股價都可以炒的漫天高，但幾天後又會跌得很慘。

**Comment by Valentinaa0214** [comment]
之前被他的廣告洗版到不行，
有載過來用，但我覺得操作有點不直覺，
不需要的時候也關不掉，後期覺得超級惱人，
常常在不必要的時候莫名跳出來，就卸載了。

**Comment by Sylvie0601** [comment]
大一英文被強迫使用Grammerly，排除那時被強迫要寫作的痛苦，我覺得Grammerly真的是好用的網站，可以修正文法、語句讓文章變得更好真的很厲害

**Comment by ru0372572** [comment]
反而是英文老師要我們一定先用Grammerly看過才能上傳作業，但可以手動關掉我是覺得還好，沒有把他卸載XD
以前覺得廣告一直:Grammerly can help. 很煩，現在發現他確實是個好用的小工具，
